views/charts.tmpl

```diff
@@ -31,6 +31,7 @@
                             <option value="ticket-price">Ticket Price</option>
                             <option value="ticket-pool-size">Ticket Pool Size</option>
                             <option value="ticket-pool-value">Ticket Pool Value</option>
+                            <option value="percent-staked-coins">Percent Staked Coins</option>
```

Comment:
For the option text, I think "Stake Participation" would be good.  It's a commonly used phrase, and the chart y-axis will clarify how stake participation is quantified.